### ISSUE TITLE
[Boss] Kel'thuzad 25man adds

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#2459]
Hello,
I'd like to report a problem that occurs only on 25-man mode of Kel'thuzad http://www.wow-mania.com/armory/?npc=15990 during phase1 when adds are coming in waves.
The way the adds are supposed to work is that abominations have normal movement speed, while 
Soul Weavers npc=16429 and npc=23563 and 
Soldiers of the Frozen Wastes npc=16427 and npc=23561 
have reduced movement speed.
This works fine as long as the adds are engaging on their own.
(PS: i dont really know why there are two different npc ids for each or whats the difference)

However, when attempting to do the achievement where you have to kill 18 abominations and begin pulling adds on purpose in addition to those engaging automatically, if you touch a Soul Weaver or Soldier of Frozen Wastes (mostly by accident), they will engage at full movement speed, rushing to your raid and possibly wiping you.
Note, this problem only happens on 25man mode, on 10man the "extra" pulls come at intended movement speed too.

TL;DR version for Keader:
Adds' movement speed breaks when being engaged by a player instead of the fight itself.

EDIT:
Slightly different but related issue > Blood DK "bloodworms" http://www.wow-mania.com/armory/?spell=49543 tend to rush among the add groups on the sides and pull the entire room for no reason (the dk is not targetting any of the adds or whatsoever)

**Comment by Dranah** [comment]
There is no change.

The weird thing is that these npcs have same NPC id for both 10man and 25man which leaves me dumb-founded how they can behave differently.
The adds in question are the 12600 health ones on 25man (Instance ID 31 according to NPC info), with NPC ID 23561.

**Comment by fvicaria** [comment]
Please re-test on dev.

**Comment by Dranah** [comment]
Still no change :(

**Comment by Keader** [comment]
test now

**Comment by Dranah** [comment]
yay works now!
